atom/browser/api/atom_api_web_contents.cc

```diff
@@ -1180,14 +1180,8 @@ void WebContents::OpenDevTools(mate::Arguments* args) {
   if (args && args->Length() == 1) {
     bool detach = false;
```

Comment:
this variable is unused now

Reply:
yep, replacing old code and then adding dep warning